src/PostManager/PostManagerInterface.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Drupal\social_post\PostManager;
 
-use Drupal\social_api\AuthManager\OAuth2ManagerInterface
+use Drupal\social_api\AuthManager\OAuth2ManagerInterface;
```

Comment:
We might need to work this topic on another issue, but AuthManager in Social Auth does not have that name because of the module's name. The name AuthManager refers to the authentication part of the module. 
If PostManager is needed, it should be changed to AuthManager and implement classes for different protocols (OAuth2, Hawk, etc.)
See: Durpal\social_auth\AuthManager\OAuth2Manager